content/how-to/audit.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Audit'
+date: '2023-12-20T12:47:09-03:00'
+categories: ["how-to"]
+description: How to use the Audit log to inspect all events in the Rhize system
+weight: 600
+menu:
+  main:
+    parent: how-to
+    identifier:
+---
+
+The _Audit Log_ provides a tamper-proof and immutable audit trail of all events that occur in the Rhize system.
+Users with appropriate permissions can access the audit log either through the UI menu or the GraphQL API.
+
+## Prerequisites
+
+To use the audit log, ensure you have the following:
+
+- If accessing to your Rhize UI environment, a user account with appropriate permissions
+- If accessing through GraphQL, you also need:
+    - The ability to [Use the Rhize GraphQL API]({{< relref "gql" >}})
+    - A token configured so that `audience` includes `audit`, and the scopes contain `audit:query`
```

Comment:
Probably worth adding a link to a page describing how to do this, and potentially noting that the `audit:query` scope should be automatically created (but not added to your users) by BaaS.

Making these changes in Keycloak is a pretty common sticking point.

Reply:
I've added a link to the setup keycloak doc, but there really needs to be another doc that explains how to configure keycloak for various common tasks. 

I've made #59 to track this.